src/renderer/components/dock/edit-resource/view.tsx

```diff
@@ -45,8 +45,8 @@ class NonInjectedEditResource extends React.Component<EditResourceProps & Depend
         const tabData = this.props.editResourceStore.getData(this.props.tab.id);
         const obj = this.resource;
 
-        if (!obj) {
-          if (store?.isLoaded) {
+        if (!obj && store) {
+          if (store.isLoaded) {
```

Comment:
So the bug was that if store was null/undefined then execution should have gone to the `else` block? These are tricky to see, and made more common by the convenience of the `?.` operator

Reply:
No the bug was that if the `store` was `null` or `undefined` nothing should have happened, but instead it did go to the `else` block.

Reply:
thanks (I meant shouldn't, corrected)